app/controllers/idv/how_to_verify_controller.rb

```diff
@@ -0,0 +1,44 @@
+module Idv
+  class HowToVerifyController < ApplicationController
+    include RenderConditionConcern
+
+    check_or_render_not_found -> { enabled? }
+
+    VERIFICATION_OPTIONS = {
+      ipp: 'ipp',
+      remote: 'remote',
+    }.freeze
+
+    def show
+      analytics.idv_doc_auth_how_to_verify_visited(**analytics_arguments)
+      @idv_how_to_verify_form = Idv::HowToVerifyForm.new
+      @verificationOptions = VERIFICATION_OPTIONS
+    end
+
+    def update
+      analytics.idv_doc_auth_how_to_verify_submitted(**analytics_arguments)
```

Comment:
You'll want to include an analytics attribute (maybe `:selection`) that indicates which choice the user made, so that you can query for stats later.